### PR TITLE
Add Hyundai Ioniq 5 Fingerprint

### PR DESCRIPTION
**Pull request by Cx01N**

### CODE REVIEW

**Comment by incognitojam** [comment]
can you share a route ID or your dongle ID?

**Comment by Cx01N** [comment]
> can you share a route ID or your dongle ID?

Dongle Id: f2a5efadd1dbf33f

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -1480,12 +1480,22 @@ class Buttons:
...
       b'\xf1\x00NE1 MFC  AT EUR RHD 1.00 1.01 99211-GI010 211007',
+      b'\xf1\x00NE1 MFC  AT USA LHD 1.00 1.01 99211-GI010 211007',
+    ],
+    (Ecu.abs, 0x7d1, None): [
+      b'\xf1\x00NE1 IEB \x07 105!\x07\x01 58520-GI000',
+      b'\xf1\x8758520GI000\xf1\x00NE1 IEB \x07 105!\x07\x01 58520-GI000'
+    ],
+    (Ecu.eps, 0x7d4, None): [
+      b'\xf1\x00NE  MDPS R 1.00 1.06 57700GI000  4NEDR106',
+      b'\xf1\x8757700GI000 \xf1\x00NE  MDPS R 1.00 1.06 57700GI000  4NEDR106'
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1498]:
```suggestion
```

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -1480,12 +1480,22 @@ class Buttons:
   CAR.IONIQ_5: {
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00NE1_ RDR -----      1.00 1.00 99110-GI000         ',
+      b'\xf1\x8799110GI000\xf1\x00NE1_ RDR -----      1.00 1.00 99110-GI000         ',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1483]:
```suggestion
```

short response already in database

File: selfdrive/car/hyundai/values.py (hunk 1 above)
Comment (sshane) [reply selfdrive/car/hyundai/values.py:1498]: as you can see from the tests failing, we only are using queries from the camera for CAN-FD cars due to a gateway bug

**Pull request merged**